### DISCUSSION TITLE
Is there a document on the RabbitMQ EXE command line variable options?

### DISCUSSION THREAD

**Discussion by DMahoney0** [rabbitmq/rabbitmq-server#7324] [Questions]
I am attempting to get a list or document on all of the command line options that exist when installing RabbitMQ.

An example:
rabbitmq-server-3.11.7.exe /S /D="C:\Program Files (x86)\RabbitMQ Server 3.11.7"

I am aware of the silent flag (/S) and the directory flag (/D). Are there others?
Specifically, I want to know if there is a flag to prevent the service from starting at the end of the installation; there is a checkbox to prevent that when installing without the silent flag.

I tried using /help, /?, and a couple of others with no luck. I also looked through the website documentation and GitHub documentation with no luck. Any information would be appreciated!

**Comment by lukebakken** [comment]
The `.exe` is built using NSIS:

https://nsis.sourceforge.io/Docs/Chapter3.html#installerusagecommon

Note that double quotes **should not be used** when using `/D`, and that argument should be the last on the command line.

Here is the full source for the installer:

https://github.com/rabbitmq/rabbitmq-packaging/blob/main/windows-exe/rabbitmq_nsi.in

`/NOSERVICERUN` will prevent the service from starting. So you should use:

```
rabbitmq-server-3.11.7.exe /NOSERVICERUN /S /D=C:\ProgramData\RabbitMQ
```
